bugsnag-android-core/src/main/java/com/bugsnag/android/BugsnagEventMapper.kt

```diff
@@ -193,15 +193,15 @@ internal class BugsnagEventMapper(
         copy["lineNumber"] = lineNumber?.toLong()
 
         (frame["frameAddress"] as? String)?.let {
-            copy["frameAddress"] = java.lang.Long.decode(it)
+            copy["frameAddress"] = it
         }
 
         (frame["symbolAddress"] as? String)?.let {
-            copy["symbolAddress"] = java.lang.Long.decode(it)
+            copy["symbolAddress"] = it
         }
 
         (frame["loadAddress"] as? String)?.let {
-            copy["loadAddress"] = java.lang.Long.decode(it)
+            copy["loadAddress"] = it
         }
```

Comment:
Maybe these should rather handle backwards compatibility with old reports that might still be in storage:

```
(frame["frameAddress"] as? Number)?.let {
    copy["frameAddress"] = "0x${it.toString(16)}"
}
```